governance-and-automation/permissions-review/create-apply-access-review-policies.md

```diff
@@ -133,9 +133,19 @@ The warning serves as a reminder that **if you have rules set up**, manually cha
 
 {% endhint %}
 
-
 ![Apply Policy Dialog - Warning](../../.gitbook/assets/create-access-review-policies-apply-policy-warning.png)
 
+{% hint style="warning" %}
+**Please note!** 
+
+Only **one Access Review policy can be active at a time** on a workspace.
+When a new Access Review policy is created and applied to workspaces, **it will be triggered depending on the date set in the Settings of the Access Review Option**.
```

Comment:
**it will be triggered depending on the date set in the Settings of the Access Review Option**.
ovaj dio mi nije jasan baš. tu se misli na Settings > Access Review Option**s**?